libraries/freertos_plus/aws/greengrass/include/aws_greengrass_discovery.h

```diff
@@ -85,7 +90,10 @@ typedef struct
  * @return If connection was successful then pdPASS is
  * returned.  Otherwise pdFAIL is returned.
  */
-BaseType_t GGD_GetGGCIPandCertificate( char * pcBuffer,
+BaseType_t GGD_GetGGCIPandCertificate( const char * pcHostAddress,
```

Comment:
For security reason, people usually suggest to add string length aside string pointer. Consider about it.

Reply:
This commit was meant mainly to update the greengrass library to remove macros included from demos headers, so no changes were made to the library besides moving these to parameters. Since including the string length along with `pcHostAddress` would require adding a field to `GGD_HostAddressData_t`, I think that change would be more appropriate when rewriting the library.

Reply:
Got it. I agree.